### PR TITLE
Add CNN

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

**Comment by Carlgo11** [comment]
Which [site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria) would it meet?

**Comment by soodaayush** [comment]
It would meet Personal Info, considering you can store your full name there.

**Comment by soodaayush** [comment]
Any reason why no comment has been left here by the twofactorauth team in weeks?

**Comment by soodaayush** [comment]
@hkamran80 @Carlgo11 Any updates on this PR?

File: img/c/cnn.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1200" height="800" viewBox="-1350 -1050 11700 6300"><path fill="#c00" d="M2800 1200h-700a900 900 0 0 0 0 1800h783a117 117 0 0 0 117-117V817a817 817 0 0 1 1526-405l874 1529V817a817 817 0 0 1 1526-405l874 1529V0h500l100 100L8500 0h500v3383a817 817 0 0 1-1526 405l-874-1529v1124a817 817 0 0 1-1526 405l-874-1529v1124a817 817 0 0 1-817 817H2100a2100 2100 0 0 1 0-4200h700v500l-100 100 100 100z"/><path fill="none" stroke="#fff" stroke-width="200" d="M2800 600h-700a1500 1500 0 0 0 0 3000h1283a217 217 0 0 0 217-217V817a217 217 0 0 1 405-108l1590 2782a217 217 0 0 0 405-108V817a217 217 0 0 1 405-108l1590 2782a217 217 0 0 0 405-108V0"/></svg>
```
Comment (hkamran80) [inline img/c/cnn.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 923.08 430.77"><path fill="#c00" d="M287.14 123.14h-71.79a92.3 92.36 0 0 0 0 184.72h80.3a12 12 0 0 0 12-12V83.83a83.79 83.84 0 0 1 156.5-41.56l89.64 156.9V83.84a83.79 83.84 0 0 1 156.5-41.56l89.63 156.9V0h51.28l10.25 10.26L871.71 0h51.28v347.16a83.79 83.84 0 0 1-156.5 41.56l-89.64-156.9v115.34a83.79 83.84 0 0 1-156.5 41.56l-89.63-156.9v115.34A83.79 83.84 0 0 1 346.93 431H215.35a215.37 215.5 0 0 1 0-431h71.8v51.31l-10.26 10.26 10.25 10.26z"/><path fill="none" stroke="#fff" stroke-width="20.52" d="M287.14 61.57h-71.79a153.83 153.93 0 0 0 0 307.86h131.58a22.25 22.27 0 0 0 22.26-22.27V83.84a22.25 22.27 0 0 1 41.53-11.08l163.06 285.48a22.25 22.27 0 0 0 41.54-11.08V83.84a22.25 22.27 0 0 1 41.53-11.08l163.07 285.48a22.25 22.27 0 0 0 41.53-11.08V0"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Am wondering if there is need for a category concerning News.

**Comment by hkamran80** [comment]
At the moment, no, considering we have such few sites that are news sites.

**Comment by soodaayush** [comment]
Made the changes.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**